test/unit/vdom/properties.js

```diff
@@ -31,14 +31,13 @@ describe('vdom/properties', () => {
 
   it('should not set properties on SVG elements', done => {
     expect(() => {
-      new (define('x-test', { // eslint-disable-line no-new
+      const Test = define('x-test', {
         render() {
-          vdom.element('svg', { height: 100 });
+          vdom.element('svg', { width: 100 });
```

Comment:
Width was the prop I had issues with but height would reproduce it too. It just got changed when I was messing with the test. It won't matter which one it is, though.